docker-compose.dev.yml

```diff
@@ -40,15 +40,20 @@ services:
     ports:
       - "${CAMPAIGN_GRPC_PORT?}:5001"
 
-  roach-lb:
-    ports:
-      - "${DB_PORT?}:26257"
-      - "3080:8080"
-      - "3081:8081"
-
   flyway:
     build:
       context: .
       dockerfile: ./flyway/Dockerfile
     volumes:
       - ./db/migrations:/flyway/sql
+
+  roach-0:
```

Comment:
If we have this configuration in `docker-compose.yml` we don't need it here as they are inherited

Reply:
yep, now moved back to the main docker-compose.yml - see commit: d3613d3
The start command for dev is "start-single-node", while in the docker-compose.prod.yml is replaced with "start"